treeherder/perf/secretary_tool.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import simplejson as json
+
+from datetime import datetime, timedelta
+from treeherder.perf.models import PerformanceSettings
+from django.conf import settings as django_settings
+
+logger = logging.getLogger(__name__)
+
+# TODO: extract strings into constants
+
+
+class SecretaryTool:
+    """
+    Tool used for doing the secretary work in the Performance Sheriff Bot.
+    """
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def manage_settings():
+        perf_sheriff_settings, created = PerformanceSettings.objects.get_or_create(
+            name="perf_sheriff_bot",
+            defaults={"settings": SecretaryTool._get_default_settings()},
```

Comment:
This extra "settings" key feels redundant. The column's name is already like that.

Reply:
it has to be provided in `defaults`, in case `get_or_create` has to create the object

Reply:
Oh, that's right.